src/Data/Skills/sup_dex.lua

```diff
@@ -2614,7 +2614,7 @@ skills["SupportInnervatePlayer"] = {
 				["support_innervate_buff_grant_%_added_lightning_attack_damage"] = {
 					mod("DamageGainAsLightning", "BASE", nil, ModFlag.Attack, 0, { type = "Condition", var = "KilledShockedLast3Seconds" }, { type = "GlobalEffect", effectType = "Buff", effectName = "Innervate" }),
 				},
-				["support_innervate_base_buff_duration"] = {
+				["support_innervate_buff_base_duration_ms"] = {
```

Comment:
I'll move it for you, but for the future, be sure to add this change to src/Export/Skills/sup_dex.txt as well, since this file is auto-generated from those.